package.json

```diff
@@ -2,6 +2,9 @@
   "name": "ng-wig",
   "version": "2.1.0",
   "author": "Stepan Suvorov",
+  "contributors": [
+    {"name": "Sascha Weniger", "url": "https://github.com/sashless"}
```

Comment:
:)

could you please then put all the names(http://stepan.js.org/ngWig/)? 
plus - https://github.com/bolelamx